package.json

```diff
@@ -134,6 +134,7 @@
     "interpolate-components": "1.1.1",
     "lodash": "^4.17.11",
     "marked": "0.5.2",
+    "prismjs": "^1.15.0",
```

Comment:
Did we need to add this to our dependencies? Isn't it a dependency of `docsify-cli` > `docsify`?

Reply:
It is, but I was still getting the `Cannot find module 'prismjs/components/prism-markup-templating'` issue when trying to run `npx docsify serve docs`, until I added this. It's probably an issue upstream, but I don't have the time to track it down right now 😕